src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -127,7 +127,16 @@ protected virtual void Initialize()
             }
 
             // Bindings may use name resolution, so provide this before reading the bindings. 
-            ScriptConfig.HostConfig.NameResolver = new NameResolver();
+            var nameResolver = new NameResolver();
+
+            var dashboardString = nameResolver.Resolve("AzureWebJobsDashboard");
```

Comment:
Instead of NameResolver, use AmbientConnectionStringProvider.Instance.GetConnectionString("Dashboard"). That is what the SDK uses, and it will also check connection strings. It will apply the "AzureWebJobs" prefix.